src/pages/WorkloadDetails/WorkloadDetailsPage.tsx

```diff
@@ -183,27 +183,17 @@ class WorkloadDetails extends React.Component<RouteComponentProps<WorkloadId>, W
           <>
             <PfTitle location={this.props.location} istio={this.state.istioEnabled} />
             <Nav bsClass="nav nav-tabs nav-tabs-pf">
-              <NavItem eventKey="info">
-                <div>Info</div>
-              </NavItem>
-              <NavItem eventKey="traffic">
-                <div>Traffic</div>
-              </NavItem>
-              <NavItem eventKey="logs">
-                <div>Logs</div>
-              </NavItem>
-              <NavItem eventKey="in_metrics">
-                <div>Inbound Metrics</div>
-              </NavItem>
-              <NavItem eventKey="out_metrics">
-                <div>Outbound Metrics</div>
-              </NavItem>
+              <NavItem eventKey="info">Overview</NavItem>
+              <NavItem eventKey="traffic">Traffic</NavItem>
+              <NavItem eventKey="logs">Logs</NavItem>
```

Comment:
is the order here OK? in other places it's traffic, then metrics, here it is traffic, logs, metrics.
(I know this existed prior to this PR, and doesn't need to be fixed as part of this PR)